test/test.js

```diff
@@ -351,4 +361,32 @@ describe('request-progress', function () {
         expect(request.progressContext).to.be.an('object');
         expect(request.progressState).to.be(null);
     });
+
+    it('should ignore "data" event from the request', function (done) {
```

Comment:
Can you change this to `should hook into "data" event from the response and not the request`